### PR TITLE
docs get post following user-id

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Docs get post following user-id
## Reference Requirements
- Task ID: #129

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-haytran** [comment]
LGTM

**Comment by at-haytran** [comment]
LGTM

File: docs/api-references/post.md [hunk 1]
```diff
@@ -179,4 +179,69 @@ Delete the post
...
+| Param | Type | Description |
+|---|---|---|
+| post_type | number | Type Of Posts |
+| page | number | Paninate Posts |
+| sort | string | Sort Posts Follow Name |
+
+```json
+{
+    "data": [
+            {
```
Comment (at-vietpham) [inline docs/api-references/post.md]: Tab 4 space thôi

File: docs/api-references/post.md [hunk 2]
```diff
@@ -179,4 +179,69 @@ Delete the post
...
+                "post_type": 1,
+                "body": "Et veniam in nemo commodi molestiae. Voluptas quibusdam perferendis beatae molestias. Qui deserunt cumque veniam commodi et debitis recusandae.",
+                "rate_point": 3.1,
+                "status": 0,
+                "deleted_at": null,
+                "created_at": "2018-06-06 08:45:01",
+                "updated_at": "2018-06-06 08:45:01",
+            }
+            ],
+"first_page_url": "http://192.168.33.10/api/books/9/posts?page=1",
```
Comment (at-vietpham) [inline docs/api-references/post.md]: Lùi vào 1 tab

**Review by ghost — CHANGES_REQUESTED** [review]

File: docs/api-references/post.md [hunk 3]
```diff
@@ -179,4 +179,69 @@ Delete the post
...
+/api/users/{user}/posts
+```
+Get list posts following user
+
+#### Query Param
+| Param | Type | Description |
+|---|---|---|
+| post_type | number | Type Of Posts |
+| page | number | Paninate Posts |
+| sort | string | Sort Posts Follow Name |
```
Comment (at-vietpham) [inline docs/api-references/post.md]: Có cho phép `limit` không?
Nhiều query như vậy cần cho ví dụ cách dùng.

**Review by ghost — CHANGES_REQUESTED** [review]

File: docs/api-references/post.md [hunk 4]
```diff
@@ -182,6 +182,79 @@ Delete the post
     }
 }
 ```
+
+### `GET` List posts following user
+```
```
Comment (at-haytran) [inline docs/api-references/post.md]: Thêm header

**Comment by at-haytran** [comment]
LGTM

**Comment by at-vietpham** [comment]
LGTM 🐙

**Pull request merged**